### DISCUSSION TITLE
Does Wix v4 have the same 2 GB filze size and 65535 file number limitations as Wix v3?

### DISCUSSION THREAD

**Discussion by DMahoney0** [wixtoolset/issues#7921] [Questions]
I've run across errors like this before:
```
light.exe : error LGHT0306: An error (E_FAIL) was returned while finalizing a CAB file. This most commonly happens when creating a CAB file with more than 65535 files in it. Either reduce the number of files in your installation package or split your installation package's files into more than one CAB file using the Media element.
```
when using Wix v3 and was curious if the same File Table limitation (2 GB and 65535 files) I've heard about [here](https://stackoverflow.com/a/18150794) still exists in Wix v4. 

I didn't see any mention of it in the Wix v4 releases notes and I suppose the same File Table limitations would remain if the operating system hasn't changed, so I imagine it does. Having confirmation of this would be appreciated.

**Comment by robmen** [comment]
The cabinet's file format maxes out at 2GB per-file, but you can span cabinets to get around that limitation. Otherwise, until Microsoft changes the Windows Installer to use a different compression container, that is a known limit.

**Comment by coolerdoncooper** [comment]
My cab file is about 2MB and I still get this error with 3.11.2. Any ideas? I've been struggling with it for about two weeks.
